src/components/Markdown.react.js

```diff
@@ -83,6 +129,12 @@ DashMarkdown.propTypes = {
          */
         component_name: PropTypes.string,
     }),
+
+    /**
+     * User-defined inline styles for the rendered Markdown
+     * Legacy from SyntaxHighlighter prop `customStyles`
```

Comment:
Not sure exactly what we used `customStyle` for previously, but this one will behave a bit differently as it's applied to the container `<div>`, not the `<pre>`. Anyway seems reasonable to include it, but let's restrict the docstring to describing what the prop does now, and put a description of its relation to `SyntaxHighlighter` in our migration guide in the docs.

Reply:
> let's restrict the docstring to describing what the prop does now
OK, done. One thing to keep in mind is that `customStyle` was one level deeper before; i.e. it was applied to the `<Markdown>` child of the parent `<div>`, not the parent `<div>` itself, as it is now. The thing is, `Markdown` isn't really a container, more of a mutator. I think there are docs examples with e.g. `style={'border-left': 'grey'}`.